CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ The development of this library follows [this very useful guide about Outside-In
 
 Semantic-release will auto-publish anything that is merged into the `next` branch to npm under the `next` dist-tag. Users can install this version by running `npm install vue-imgix@next`. To facilitate this, every commit must follow the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0-beta.2/).
 
-The current process to publish the master dist-tag from the master branch:
+The current process to publish the main dist-tag from the main branch:
```

Comment:
Unrelated question: do we want to include publishing instructions if it's not something contributing users will have to worry about? That might be better suited for internal docs imo.

Reply:
No strong opinions here. Weak opinion that it's better to keep docs as close as possible to the repo so that it's self-contained, i.e. so the repo is a self-contained item with documentation of how to build/release it... but happy to go either way. What about you?

Reply:
Hmm yeah, that's a pretty good point I hadn't considered. I think so long as it's kept separate from the README (like it is here), then I don't mind.